cmake_modules/optionDefaults.cmake

```diff
@@ -4,7 +4,12 @@ endif()
 
 if(NOT PREFIX)
     if(WIN32)
-        set(PREFIX "$ENV{ProgramFiles}")
+        set(prog86key "ProgramFiles(x86)")
+        if("$ENV{${prog86key}}" STREQUAL "")
```

Comment:
Shouldn't this be determined by the package being built rather than the environment?

IOW When building the 32bit package, you always want it to install into the 32bit ProgFiles (and ditto for the 64bit).